pkg/karmadactl/get.go

```diff
@@ -188,6 +200,7 @@ type OtherPrint struct {
 }
 
 // Run performs the get operation.
+//nolint:gocyclo
```

Comment:
Is there a plan to remove this `nolint:gocyclo`?

Reply:
Ok, I will do it in this pr.

Reply:
Removing this `nolint:gocyclo` need to split code that make code unreadable and complicated, so not removed for now and continue to find a good way.

Reply:
I'm ok with it. Could you please file an issue for this? You know, the technical debt is pretty easy to forget.

Reply:
Ok, add a issue that list all `nolint:gocyclo` in codes as items and plan to remove them?

Reply:
Not list all, just focus on the new one would be ok.